charts/dgraph/README.md

```diff
@@ -218,14 +222,25 @@ The following table lists the configurable parameters of the `dgraph` chart and
 | `global.ingress.tls`                     | global ingress tls settings                                           | `{}`                                                |
 | `global.ingress.ratel_hostname`          | global ingress virtual host name for Ratel service                    | `""`                                                |
 | `global.ingress.alpha_hostname`          | global ingress virtual host name for Alpha service                    | `""`                                                |
+| `global.ingress_grpc.enabled`             | Enable global ingress-grpc resource (overrides Alpha ingress-grpc)   | `false`                                             |
+| `global.ingress_grpc.annotations`         | global ingress-grpc annotations                                      | `{}`                                                |
+| `global.ingress_grpc.tls`                 | global ingress-grpc tls settings                                     | `{}`                                                |
+| `global.ingress_grpc.alpha_grpc_hostname` | global ingress-grpc virtual host name for Alpha GRPC service         | `""`                                                |
+## Ingress resource
 
+You can define ingress resources through `alpha.ingress` for the Alpha HTTP(S) service and `ratel.ingress` for the ratel UI service, or you can use a combined single ingress with `global.ingress` for both Alpha HTTP(S) and ratel UI services.
 
-## Ingress resource
+There are some example chart values for ingress resource configuration in [example_values/ingress](https://github.com/dgraph-io/charts/tree/master/charts/dgraph/example_values/ingress).
+### Ingress resource with GRPC
 
-You can define ingress resources through `alpha.ingress` for the Alpha HTTP service and `ratel.ingress` for the ratel UI service, or you can use a combined single ingress with `global.ingress` for both Alpha HTTP and ratel UI services.
+The Dgraph Alpha GRPC can be exposed though an ingress configured with either `alpha.ingress_grpc` or `global.ingress_grpc` values.  For this solution to work, a few requirements must be met:
 
-There are some example chart values for ingress resource configuration in [example_values](https://github.com/dgraph-io/charts/tree/master/charts/dgraph/example_values).
+* if an external load balancer is used by the ingress controller, the load balancer should be Layer 4 TCP as the ingress-controller will handle both HTTPS and GRPC traffic.
+* The ingress resource must instruct the ingress controller to use GRPC, such as `nginx.ingress.kubernetes.io/backend-protocol: GRPC` with [ingress-nginx](https://kubernetes.github.io/ingress-nginx/).
+* The ingress controller can support both HTTPS and GRPC traffic, but must do TLS terminationation, so a secret with the appropriate certificates must be configured; this can be automated with the [cert-manager](https://cert-manager.io/) addon.
```

Comment:
```suggestion
* The ingress controller can support both HTTPS and GRPC traffic, but must do TLS termination, so a secret with the appropriate certificates must be configured; this can be automated with the [cert-manager](https://cert-manager.io/) addon.
```